lib/config/fileconf.go

```diff
@@ -417,10 +417,6 @@ type Auth struct {
 	// relative to the global data dir
 	LicenseFile string `yaml:"license_file,omitempty"`
 
-	// FOR INTERNAL USE:
-	// Authorities : 3rd party certificate authorities (CAs) this auth service trusts.
-	Authorities []Authority `yaml:"authorities,omitempty"`
```

Comment:
This config field is obsolete and no longer used.
Removed it instead of migrating to new schema.